examples/resources/elasticstack_kibana_prebuilt_rule/import.sh

```diff
@@ -0,0 +1 @@
+terraform import elasticstack_kibana_prebuilt_rule.enable "default"
```

Comment:
The resource doesn't implement an import handler, so this command will fail.

IMO I'm not sure import actually makes sense for this resource though, it's quite difficult for us to know that all rules matching a given tag set have been enabled. I don't think there's a real impact to just running create again if the rules have been installed and the expected rules are already enabled?